drillsrs/question.py

```diff
@@ -0,0 +1,18 @@
+from drillsrs import util
+
+
+def render_question(raw_question, raw_answers, tags):
+    question_header_template = "Question: %s"
+
+    if util.is_raw_multiline(raw_question):
+        print(
+            question_header_template
+            % ("[%s]" % util.format_card_tags(tags) if tags else "")
+        )
+        return util.from_raw_to_multiline(raw_question)
```

Comment:
Now this both prints and returns a string

Reply:
I'm back! I've tried to solve it by putting everything in a single string

> Now this both prints and returns a string, it should do either one thing or another

This wasn't noticeable in the test runs I did... My bad